crowdin.yml

```diff
@@ -1,3 +1,4 @@
 files:
   - source: /src/main/resources/net/rptools/maptool/language/i18n.properties
     translation: /%original_path%/%file_name%_%locale_with_underscore%.%file_extension%
+    escape_quotes: 1
```

Comment:
This should be `3` otherwise parameterless strings like `lineParser.rollOptionComma` will start showing with doubled single quotes.

Reply:
done